content/blog/setting-up-github-actions-for-v.md

```diff
@@ -0,0 +1,355 @@
+---
+title: "Setting Up GitHub Actions for V"
+subtitle: "Introducing **Setup V**, and discussing why CI/CD is important."
+summary: "This article will walk you through setting up Github Actions for V. We begin with a primer on CI/CD, the various benefits and drawbacks. Then we make use of the Setup V action to set up a CI/CD environment for V, how to set it up, and how to use it. We will close with a link to a more sophisticated example."
+page: Blog
+authorname: Subhomoy Haldar
+authorlink: subhomoy-haldar
+categories: ["tutorials"]
+date: 2022-03-14T12:28:35+05:30
+image:
+---
+
+## The Article Series
+
+> 1. [The Complete Beginner’s Guide to CLI Apps in V](/the-complete-beginners-guide-to-cli-apps-in-v/)
+> 2. [Elevate Your V Project With Unit Tests](/elevate-your-v-project-with-unit-tests/)
+> 3. **Setting Up GitHub Actions for V** _(You are here)_
+
+## What is CI/CD, DevOps, etc.?
+
+Software Development is a very involved process, with a lot of moving parts.
+This is especially true when you have several developers working in one or
+more teams. The ultimate objective of these endeavours is to understand the
+client requirements, make sure everybody is on the same page, and eventually,
+deliver the final product.
+
+**Continuous Integration** for the most part is about the second item in the
+list: _making sure everyone is on the same page_. Individuality is important
+for problem solving: it offers new angles, allows people to shine where they
+are good at. But it can be a problem in case of things like formatting
```

Comment:
```suggestion
for problem solving: it offers new angles and allows people to shine where they
are able. But it can be a problem in case of things like formatting
```